Extensions/PrimitiveDrawing/Extension.cpp

```diff
@@ -757,6 +757,55 @@ void DeclarePrimitiveDrawingExtension(gd::PlatformExtension& extension) {
       .AddParameter("object", _("Object"), "Drawer")
       .AddParameter("expression", _("X drawing position"))
       .AddParameter("expression", _("Y drawing position"));
+  obj.AddAction("SetAntialiasing",
+                _("Set anti-aliasing on the shape drawer"),
+                _("(De)Activate anti-aliasing on the shape drawer"),
+                _("Set anti-aliasing of _PARAM0_ to _PARAM1_"),
+                _("Setup"),
+                "res/actions/antialiasing24.png",
+                "res/actions/antialiasing.png")
+      .AddParameter("object", _("Object"), "Drawer")
+      .AddParameter(
+          "yesorno", _("Activated ?"), "Is the antialiasing activated ?");
+  obj.AddCondition(
+         "IsAntialiasingOn",
+         _("Anti-aliasing activated on the shape drawer"),
+         _("Checks if the anti-aliasing is activated for a shape drawer"),
+         _("Is the anti-aliasing enabled on _PARAM0_ ?"),
+         _("Setup"),
+         "res/actions/antialiasing24.png",
+         "res/actions/antialiasing.png")
+      .AddParameter("object", _("Object"), "Drawer");
 
+  obj.AddAction("SetAntialiasingQuality",
+                _("Anti-aliasing quality"),
+                _("Anti-aliasing quality"),
+                _("Set antialiasing quality of _PARAM0_ to _PARAM1_"),
+                _("Setup"),
+                "res/actions/antialiasing24.png",
+                "res/actions/antialiasing.png")
+      .AddParameter("object", _("Object"), "Drawer")
+      .AddParameter("stringWithSelector",
+                    _("Antialiasing quality level"),
```

Comment:
Why not add a "None" quality instead of using 2 actions set an anti-aliasing?

Reply:
I like this idea. How about "Off/Low/Medium/High"?

Reply:
There already is an event to turn off/on the antialising. Do you think I should merge all into one action ?